code/src/WijmoProvider/Grid/FlexGrid.ts

```diff
@@ -186,6 +212,67 @@ namespace WijmoProvider.Grid {
                 }
             }
         }
+        /**
+         * Mark a group of Data Grid lines with given keys (from the KeyBinding field) as saved in the database.
+         *
+         * @param {Array<string>} rowKeys List of row identifiers on the KeyBinding field.
+         * @param {boolean} forceClearValidationMarks determines whether or not we should clean the validation marks.
+         * @memberof FlexGrid
+         */
+        public clearAllChangesByRowKeys(
+            rowKeys: Array<string>,
+            forceClearValidationMarks: boolean
+        ): void {
+            //if the row keys array is empty we will throw an error
+            if (rowKeys.length === 0) {
+                throw new Error(OSFramework.Enum.ErrorMessages.Row_EmptyList);
+            }
+            //if the row keys array as empty keys we will throw an error
+            if (
+                rowKeys.indexOf('') > -1 ||
+                rowKeys.indexOf(null) > -1 ||
+                rowKeys.indexOf(undefined) > -1
+            ) {
+                throw new Error(
+                    OSFramework.Enum.ErrorMessages.Row_ListEmptyValues
```

Comment:
Missing ErrorCode?

Reply:
@joselrio The error codes are set at the API level.